### ISSUE TITLE
Reconfiguring goal tolerances for topo nav end nodes

### ISSUE THREAD

**Issue by cdondrup** [strands-project/strands_navigation#235]
I have quite a particular question, I would like to be able to reconfigure the yaw and maybe also xy goal tolerance for the goal node of topological navigation. Here is the reason, at AAF I have to use intermediate nodes for the walking group, where the robot shouldn't do anything. These nodes are just used to define the rout because we do have two big loops in there and I want the robot to traverse specific parts of the environment. When reaching such a node, the robot should neither really stop nor do anything but just continue to the next node. In our environment we do have a lot of nodes that I could use but, thanks to the bellbot, they all face doors, so the robot would turn 90 degrees before continuing to drive on. This is not very legible or desirable. I only need the nodes to define the rout more precisely. If I could reconfigure the yaw goal tolerance that would solve that problem. Also I don't care if the robot reaches the node precisely, so reconfiguring the xy would be a cheap way of allowing for the node to be blocked.

I understand that there is a feature where I can specify that a node should always have a specific yaw tolerance, but than that would also apply when the walking group is not running which the bellbot people won't like.

Long story short, what would be the best and easiest way to dynamically reconfigure the yaw and xy goal tolerance for topo nav?

Additionally, would it be useful to be able to have topo nav plan a rout given constraints like passing through certain nodes? Is that theoretically possible?

Maybe @Jailander or @bfalacerda have an opinion on that.

**Comment by Jailander** [comment]
Well in nodes that are only used for the walking group as final goals you can set this value  https://github.com/strands-project/strands_navigation/blob/indigo-devel/strands_navigation_msgs/msg/TopologicalNode.msg#L6 on the nodes definition, for other cases I'm not sure how to tackle it, maybe we can activate a mode in topological navigation that doesn't care about orientation at goals setting a parameter or something, but I don't know if it is an overkill.

**Comment by Jailander** [comment]
@cdondrup can we close this?? #236 was made for this issue

**Comment by cdondrup** [comment]
Yes. Thanks!
